skcosmo/datasets/_base.py

```diff
@@ -0,0 +1,32 @@
+from os.path import join, dirname
+import numpy as np
+from sklearn.utils import Bunch
+
+
+def load_degenerate_manifold():
+    """Load and return the degenerate manifold dataset.
+
+    Returns
+    -------
+    degenerate_manifold_data : sklearn.utils.Bunch
+        Dictionary-like object, with the following attributes:
+
+        data : `sklearn.utils.Bunch` --
+        contains the keys ``SOAP_power_spectrum`` and ``SOAP_bispectrum``.
+        Two representations of the carbon environments of the
+        degenerate manifold dataset.
+
+        DESCR: `str` --
+        The full description of the dataset.
+    """
+    module_path = dirname(__file__)
+    target_filename = join(module_path, "data", "degenerate_manifold.npz")
```

Comment:
Is this file added to the sdist when packaging skcosmo? We may have to specify the `data_file` list in `setup.cfg`. If you run `python setup.py sdist`, is the file included?

Reply:
I guess it is, otherwise the test would fail?

Reply:
If I remove the file for me locally they fail.

Reply:
That's strange, because the file is not included (meaning it would not be part of the uploaded package on PyPI). Here is my log running on your branch:

```
$ python setup.py sdist
running sdist
running egg_info
writing skcosmo.egg-info/PKG-INFO
writing dependency_links to skcosmo.egg-info/dependency_links.txt
writing requirements to skcosmo.egg-info/requires.txt
writing top-level names to skcosmo.egg-info/top_level.txt
reading manifest file 'skcosmo.egg-info/SOURCES.txt'
writing manifest file 'skcosmo.egg-info/SOURCES.txt'
running check
warning: check: missing required meta-data: url

warning: check: missing meta-data: either (author and author_email) or (maintainer and maintainer_email) must be supplied

creating skcosmo-0.1.0
creating skcosmo-0.1.0/skcosmo
creating skcosmo-0.1.0/skcosmo.egg-info
creating skcosmo-0.1.0/skcosmo/datasets
copying files to skcosmo-0.1.0...
copying LICENSE -> skcosmo-0.1.0
copying README.md -> skcosmo-0.1.0
copying pyproject.toml -> skcosmo-0.1.0
copying setup.cfg -> skcosmo-0.1.0
copying setup.py -> skcosmo-0.1.0
copying skcosmo/__init__.py -> skcosmo-0.1.0/skcosmo
copying skcosmo.egg-info/PKG-INFO -> skcosmo-0.1.0/skcosmo.egg-info
copying skcosmo.egg-info/SOURCES.txt -> skcosmo-0.1.0/skcosmo.egg-info
copying skcosmo.egg-info/dependency_links.txt -> skcosmo-0.1.0/skcosmo.egg-info
copying skcosmo.egg-info/requires.txt -> skcosmo-0.1.0/skcosmo.egg-info
copying skcosmo.egg-info/top_level.txt -> skcosmo-0.1.0/skcosmo.egg-info
copying skcosmo.egg-info/zip-safe -> skcosmo-0.1.0/skcosmo.egg-info
copying skcosmo/datasets/__init__.py -> skcosmo-0.1.0/skcosmo/datasets
copying skcosmo/datasets/_base.py -> skcosmo-0.1.0/skcosmo/datasets
Writing skcosmo-0.1.0/setup.cfg
creating dist
Creating tar archive
removing 'skcosmo-0.1.0' (and everything under it)
```

```
$ tar tfv dist/skcosmo-0.1.0.tar.gz
drwxr-xr-x  0 guillaume staff       0 19 nov 18:42 skcosmo-0.1.0/
-rw-r--r--  0 guillaume staff    1529 25 oct 17:21 skcosmo-0.1.0/LICENSE
-rw-r--r--  0 guillaume staff    2400 19 nov 18:42 skcosmo-0.1.0/PKG-INFO
-rw-r--r--  0 guillaume staff    1744 26 oct 10:38 skcosmo-0.1.0/README.md
-rw-r--r--  0 guillaume staff     435 19 nov 18:13 skcosmo-0.1.0/pyproject.toml
-rw-r--r--  0 guillaume staff     279 19 nov 18:42 skcosmo-0.1.0/setup.cfg
-rw-r--r--  0 guillaume staff     239 19 nov 18:13 skcosmo-0.1.0/setup.py
drwxr-xr-x  0 guillaume staff       0 19 nov 18:42 skcosmo-0.1.0/skcosmo/
-rw-r--r--  0 guillaume staff      22 18 nov 11:16 skcosmo-0.1.0/skcosmo/__init__.py
drwxr-xr-x  0 guillaume staff       0 19 nov 18:42 skcosmo-0.1.0/skcosmo/datasets/
-rw-r--r--  0 guillaume staff      92 19 nov 18:41 skcosmo-0.1.0/skcosmo/datasets/__init__.py
-rw-r--r--  0 guillaume staff    1070 19 nov 18:41 skcosmo-0.1.0/skcosmo/datasets/_base.py
drwxr-xr-x  0 guillaume staff       0 19 nov 18:42 skcosmo-0.1.0/skcosmo.egg-info/
-rw-r--r--  0 guillaume staff    2400 19 nov 18:42 skcosmo-0.1.0/skcosmo.egg-info/PKG-INFO
-rw-r--r--  0 guillaume staff     306 19 nov 18:42 skcosmo-0.1.0/skcosmo.egg-info/SOURCES.txt
-rw-r--r--  0 guillaume staff       1 19 nov 18:42 skcosmo-0.1.0/skcosmo.egg-info/dependency_links.txt
-rw-r--r--  0 guillaume staff      14 19 nov 18:42 skcosmo-0.1.0/skcosmo.egg-info/requires.txt
-rw-r--r--  0 guillaume staff       8 19 nov 18:42 skcosmo-0.1.0/skcosmo.egg-info/top_level.txt
-rw-r--r--  0 guillaume staff       1 19 nov 18:41 skcosmo-0.1.0/skcosmo.egg-info/zip-safe
```

Reply:
I think we need to add a `MANIFEST.in` file to include such extra data files